tests/test_upload.py

```diff
@@ -348,12 +348,16 @@ def test_prints_skip_message_for_uploaded_package(
     assert result is None
 
     captured = capsys.readouterr()
-    assert "Skipping twine-1.5.0-py2.py3-none-any.whl" in captured.out
     assert RELEASE_URL not in captured.out
 
+    assert caplog.messages == [
+        "Skipping twine-1.5.0-py2.py3-none-any.whl "
+        "because it appears to already exist"
+    ]
```

Comment:
Turns out there was a test bug because `test_upload.py` runs after `test_main.py`, and therefore was using the Rich output configuration. The new fixture in `conftest.py` normalizes the behavior.